tools/dataMigration/migrateUsersRole.ts

```diff
@@ -76,6 +76,7 @@ export const migrateUsersRole = async (props: Props): Promise<void> => {
                    else '{}'::jsonb
                    end
                                                                                                           as roles,
+               '{}'::jsonb                                                                                as info,
```

Comment:
this will be needed in the next pr, and use column names props and permissions

Reply:
my bad, wrong file